From 6f95a37c1dd600f05b6678d1c142d38050eddf73 Mon Sep 17 00:00:00 2001
Message-Id: <6f95a37c1dd600f05b6678d1c142d38050eddf73.1351081950.git.crobinso@redhat.com>
From: Marc Deslauriers <marc.deslauriers@canonical.com>
Date: Wed, 22 Aug 2012 13:25:25 -0400
Subject: [PATCH 1/3] virt-manager: Fix KVM_PACKAGES substitution variable

Hello,

Commit 68e78244e851f1f3829f711249d1f375103a2a4d changed the HV_PACKAGES
variable to KVM_PACKAGES in src/virt-manager.py.in, but the
corresponding change was never made to src/Makefile.am.

Here is a patch to correct the issue.

Thanks,

Marc.

commit 93ba787355736733c189e6809b9108a70f0e02d4
Author: Marc Deslauriers <marc.deslauriers@ubuntu.com>
Date:   Wed Aug 22 13:21:48 2012 -0400

    Fix incorrect substitution variable
(cherry picked from commit 2f2ba0acc0a3a3c1a45cd581495f185983f87772)
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index e51d7b2..3f4ee8f 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -65,7 +65,7 @@ endif
 	  -e "s,::DEFAULT_QEMU_USER::,$(DEFAULT_QEMU_USER)," \
 	  -e "s,::ENABLE_UNSUPPORTED_RHEL_OPTS::,$(ENABLE_UNSUPPORTED_RHEL_OPTS)," \
 	  -e "s|::PREFERRED_DISTROS::|$(PREFERRED_DISTROS)|" \
-	  -e "s|::HV_PACKAGES::|$(KVM_PACKAGES)|" \
+	  -e "s|::KVM_PACKAGES::|$(KVM_PACKAGES)|" \
 	  -e "s|::ASKPASS_PACKAGE::|$(ASKPASS_PACKAGE)|" \
 	  -e "s|::LIBVIRT_PACKAGES::|$(LIBVIRT_PACKAGES)|" \
 	  < $< > $@
-- 
1.7.11.7

